pkg/kubecost/config.go

```diff
@@ -34,25 +38,32 @@ type LabelConfig struct {
 
 // NewLabelConfig creates a new LabelConfig instance with default values.
 func NewLabelConfig() *LabelConfig {
-	return &LabelConfig{
-		DepartmentLabel:          "department",
-		EnvironmentLabel:         "env",
-		OwnerLabel:               "owner",
-		ProductLabel:             "app",
-		TeamLabel:                "team",
-		ClusterExternalLabel:     "kubernetes_cluster",
-		NamespaceExternalLabel:   "kubernetes_namespace",
-		ControllerExternalLabel:  "kubernetes_controller",
-		DaemonsetExternalLabel:   "kubernetes_daemonset",
-		DeploymentExternalLabel:  "kubernetes_deployment",
-		StatefulsetExternalLabel: "kubernetes_statefulset",
-		ServiceExternalLabel:     "kubernetes_service",
-		PodExternalLabel:         "kubernetes_pod",
-		DepartmentExternalLabel:  "kubernetes_label_department",
-		EnvironmentExternalLabel: "kubernetes_label_env",
-		OwnerExternalLabel:       "kubernetes_label_owner",
-		ProductExternalLabel:     "kubernetes_label_app",
-		TeamExternalLabel:        "kubernetes_label_team",
+	if env.GetLabelConfigPath() != "" {
+		labelConfig := LabelConfig{}
+		bytes, _ := ioutil.ReadFile(env.GetLabelConfigPath())
```

Comment:
Hi @tomkennes , thank you so much for this PR! You should be returning the default configs on any errors (the config on the else case, instead of suppressing them.